.circleci/config.yml

```diff
@@ -129,6 +129,16 @@ jobs:
             docker push usdotfhwaops/v2xhubamd:${DOCKER_TAG}
             docker push usdotfhwaops/php:${DOCKER_TAG}
             docker push usdotfhwaops/port-drayage-webservice:${DOCKER_TAG}
+  docker_build_ubuntu22:
```

Comment:
Is there any way we could use matrix for ubuntu versions https://circleci.com/docs/using-matrix-jobs/

Reply:
Because the jobs are not quite equivalent I think this is a bit more complex than we want to do right now.  Also I think when we get to doing multi-architecture images it will be a different setup anyhow,